### PR TITLE
x11-apps/: cleanup

### PR DESCRIPTION
**Pull request by Moiman**
@SoapGentoo

### CODE REVIEW

File: x11-apps/radeontop/radeontop-1.0.ebuild [hunk 1]
```diff
@@ -14,18 +14,19 @@ KEYWORDS="~amd64 ~x86"
...
 
 RDEPEND="
-	sys-libs/ncurses
-	x11-libs/libpciaccess
+	sys-libs/ncurses:0=
 	x11-libs/libdrm
-	nls? ( sys-libs/ncurses[unicode] virtual/libintl )
+	x11-libs/libpciaccess
+	x11-libs/libxcb
+	nls? ( sys-libs/ncurses:0=[unicode] virtual/libintl )
```
Comment (SoapGentoo) [inline x11-apps/radeontop/radeontop-1.0.ebuild]: put this on two lines for readability

File: x11-apps/radeontop/radeontop-1.0.ebuild [hunk 2]
```diff
@@ -14,18 +14,19 @@ KEYWORDS="~amd64 ~x86"
...
 DEPEND="${RDEPEND}
 	virtual/pkgconfig
 	nls? ( sys-devel/gettext )
 "
 
 src_prepare() {
-	epatch_user
+	default
 
 cat > include/version.h << EOF || die
```
Comment (SoapGentoo) [inline x11-apps/radeontop/radeontop-1.0.ebuild]: use the tab-version of cat: http://stackoverflow.com/questions/19734083/indenting-code-when-redirecting-text
this makes everything more readable and properly indented

File: x11-apps/radeontop/radeontop-9999.ebuild [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
```
Comment (SoapGentoo) [inline x11-apps/radeontop/radeontop-9999.ebuild]: same here obviously

File: x11-apps/scripts/scripts-1.0.1-r1.ebuild [hunk 4]
```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=3
+EAPI=5
 
 XORG_STATIC="no"
```
Comment (SoapGentoo) [inline x11-apps/scripts/scripts-1.0.1-r1.ebuild]: in `src_install`:
`rm "${ED%/}"/usr/bin/xauth_switch_to_sun-des-1 || die`

also free line before `RDEPEND`

File: x11-misc/xpad/xpad-4.2.ebuild [hunk 5]
```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
 inherit eutils
```
Comment (SoapGentoo) [inline x11-misc/xpad/xpad-4.2.ebuild]: what is this needed for? also check whether `DOCS` is really needed, as EAPI 5 already has the default `DOCS` set in `src_install`

File: x11-apps/radeontop/radeontop-1.0.ebuild (hunk 2 above)
Comment (Moiman) [inline x11-apps/radeontop/radeontop-1.0.ebuild]: Should I handle the following QA issue? It seems very minor one.
 * QA Notice: The following shared libraries lack a SONAME
 * /usr/lib/libradeontop_xcb.so

Comment (SoapGentoo) [inline x11-apps/radeontop/radeontop-1.0.ebuild]: if you like, you'll need to add stuff to the link line

**Pull request closed without merging**